### PR TITLE
[AIRFLOW-1731] Set pythonpath for logging

### PR DESCRIPTION
**Pull request by Fokko**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

Before initializing the logging framework, we want to set the python path so the logging config can be found.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-1731

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: UPDATING.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ The main benefit is easier configuration of the logging by setting a single cent
 logging_config_class = my.path.default_local_settings.LOGGING_CONFIG
 ```
 
-The logging configuration file that contains the configuration needs te on the  the `PYTHONPATH`, for example in `~/airflow/dags` or `~/airflow/plugins`. These directories are loaded by default, of course you are free to add a directory to the `PYTHONPATH`, this might be handy when you have the config in another directory or you mount a volume in case of Docker. As an example you can start from `airflow.config_templates.airflow_local_settings.LOGGING_CONFIG`:
+The logging configuration file that contains the configuration needs te on the  the `PYTHONPATH`, for example `$AIRFLOW_HOME/config`. This directory is loaded by default. Of course you are free to add any directory to the `PYTHONPATH`, this might be handy when you have the config in another directory or you mount a volume in case of Docker.
```
Comment (saguziel) [inline UPDATING.md]: `needs te on the the`  ???

Comment (Fokko) [reply UPDATING.md]: Thanks, something went wrong there

**Comment by bolkedebruin** [comment]
LGTM, we should also start focusing on some documentation for this. UPDATING.md by itself isn't enough.

**Comment by ahvigil** [comment]
Yes, new logging system needs documentation-- It's currently very unintuitive. For example, `logging_config_class` is not a class at all, but a dictionary...

**Pull request closed without merging**